cloud_governance/cost_expenditure/generate_cost_explorer_report.py

```diff
@@ -0,0 +1,79 @@
+import json
+from multiprocessing import Process
+
+from cloud_governance.common.aws.cost_explorer.cost_explorer_operations import CostExplorerOperations
+from cloud_governance.common.logger.init_logger import logger
+from cloud_governance.common.elasticsearch.elasticsearch_operations import ElasticSearchOperations
+
+
+class GenerateCostExplorerReport:
+
+    def __init__(self, es_host: str, es_port: str, es_index: str, metric_type: str):
```

Comment:
Can someone run cost explorer and extract the data to file and not to ES ?

Reply:
yes. By giving full file_path.

Reply:
So please change the es fields to be empty by default 
es_host: str= '', es_port: str= '', es_index: str='', metric_type: str='BlendedCost'